### PR TITLE
ADDER software paper

### PR DESCRIPTION
**Pull request by ac-freeman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -569,7 +569,7 @@ arXiv preprint arXiv, 2021.
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
 - <a name="Freeman23arxiv"></a>Freeman, A., Mayer-Patel, K., Singh, M.,  
 *[Accelerated Event-Based Feature Detection and Compression for Surveillance Video Systems](https://arxiv.org/pdf/2312.08213.pdf)*,  
-arXiv 2023.
+ACM Multimedia Systems (MMSys), 2024. [Code](https://github.com/ac-freeman/adder-codec-rs)
```
Comment (guillermogb) [inline README.md:572]: Is there an official DOI link from ACM?

File: README.md [hunk 2]
```diff
@@ -2881,10 +2881,12 @@ Front. Neurosci. (2020), 13:1338. [Code](https://github.com/neuromorphic-paris/t
...
 - [aedat-rs](https://github.com/ac-freeman/aedat-rs) Standalone Rust library for decoding AEDAT 4 files for use in bespoke Rust event systems.
 - [expelliarmus](https://github.com/fabhertz95/expelliarmus) A pip-installable Python library to decode DAT, EVT2 and EVT3 files generated by Prophesee cameras to structured NumPy arrays.
-- [ADΔER](https://github.com/ac-freeman/adder-codec-rs/) A suite of tools for transcoding, inspecting, visualizing, and building applications for a unified intensity event representation
+- [ADΔER](https://github.com/ac-freeman/adder-codec-rs/) A suite of tools for transcoding, inspecting, visualizing, lossy compressing, and building applications for a unified intensity event representation. Supports iniVation, Prophesee, and frame-based video sources.
   - <a name="Freeman23mmsys"></a>Freeman, A.,  
     *[The ADΔER Framework: Tools for Event Video Representations](https://doi.org/10.1145/3587819.3593028)*,  
     ACM Multimedia Systems (MMSys) Doctoral Symposium, 2023. [PDF](https://dl.acm.org/doi/pdf/10.1145/3587819.3593028), [Code](https://github.com/ac-freeman/adder-codec-rs/).
+  - <a name="FreemanOpenSource"></a>Freeman, A.,
```
Comment (guillermogb) [inline README.md:2888]: - Include year in the key?
- Is there an official publication venue?

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ac-freeman) [reply README.md:572]: There won't be until April

File: README.md (hunk 2 above)
Comment (ac-freeman) [reply README.md:2888]: I added arXiv and the year. It's currently under review, so I'll update it if it gets accepted.

**Review by guillermogb — APPROVED** [review]
thanks

**Pull request merged**